about.html

```diff
@@ -0,0 +1,110 @@
+
+  <!DOCTYPE html>
+  <html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <!--Google Fonts-->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Fredoka+One&family=Lato:ital,wght@0,300;0,400;0,700;0,900;1,300;1,400;1,700;1,900&display=swap"
+      rel="stylesheet">
+    <script src="https://kit.fontawesome.com/681ec73d24.js" crossorigin="anonymous"></script>
+    <title>Oliver's Capstone Project</title>
+  </head>
+  <body>
+    <!--Menu-->
+    <header>
+      <nav class="language-navbar">
+        <ul class="navMenu">
+          <li class="lang-nav-item"><a href="#"><i class="fab fa-facebook-f"></i></a></li>
+          <li class="lang-nav-item"><a href="#"><i class="fab fa-twitter"></i></a></li>
+          <li class="lang-nav-item"><a class="top-link">English</a></li>
+          <li class="lang-nav-item"><a class="top-link">My Page</a></li>
+          <li class="lang-nav-item"><a class="top-link">Logout</a></li>
+        </ul>  
+      </nav>
```

Comment:
- [ ] Please note that on the `about` page mobile screen, there is a little overflow to the right. Kindly fix that for a better user interface😉
![screenshot (33)](https://user-images.githubusercontent.com/44624138/143656781-52c0bc6c-8f04-472c-8da0-ff88b4d1e3cc.png)